int32 trk_idx
float32 trk_x
float32 trk_y
float32 trk_vx
float32 trk_vy